lib/browser/guest-view-manager.js

```diff
@@ -48,6 +47,13 @@ const supportedWebViewEvents = [
 const guestInstances = {}
 const embedderElementsMap = {}
 
+function sanitizeOptionsForGuest (options) {
+  const ret = { ...options }
```

Comment:
what about?
```js
// WebContents values can't be sent over IPC.
const { webContents, ...ret } = options
return ret
```

Reply:
I think that'd cause an 'unused variable' warning...